test/connect.js

```diff
@@ -378,11 +378,11 @@ describe('connect', function() {
 			});
 		});
 
-		it('should update inner component when pure component\'s prop values change', function(done) {
+		it('should update inner component when pure component\'s prop values change', () => {
```

Comment:
The functions that were using the `done` callback still need to receive it, right? This should probably be `(done) => {` instead.